infrastructure/cli/cmd/create/helm.go

```diff
@@ -0,0 +1,218 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	watch "k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"log"
+)
+
+const airyYamlConfigMap = "airy-yaml-config"
+const serviceAccountName = "helm-account"
+
+const defaultAiryConfig = `
+global:
+  appImageTag: develop
+  containerRegistry: ghcr.io/airyhq
+  namespace: default
+`
+
+type Helm struct {
+	name      string
+	version   string
+	namespace string
+	clientset *kubernetes.Clientset
+}
+
+func New(kubeConfigPath string, version string, namespace string) Helm {
+	config, err := clientcmd.BuildConfigFromFlags("", kubeConfigPath)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	clientSet, clientSetErr := kubernetes.NewForConfig(config)
+	if clientSetErr != nil {
+		log.Fatal(clientSetErr)
+	}
+
+	return Helm{
+		name:      "helm-runner",
+		namespace: namespace,
+		version:   version,
+		clientset: clientSet,
+	}
+}
+
+func (h *Helm) Setup() error {
+	accountClient := h.clientset.CoreV1().ServiceAccounts(h.namespace)
+
+	account := &corev1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      serviceAccountName,
+			Namespace: h.namespace,
+		},
+	}
+	_, err := accountClient.Create(context.TODO(), account, metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+
+	roleBindingClient := h.clientset.RbacV1().ClusterRoleBindings()
+
+	roleBinding := &rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "helm-account-binding",
+			Namespace: h.namespace,
+		},
+		Subjects: []rbacv1.Subject{{
+			Namespace: h.namespace,
+			Kind:      "ServiceAccount",
+			Name:      serviceAccountName,
+		}},
+		RoleRef: rbacv1.RoleRef{
+			Kind: "ClusterRole",
+			Name: "cluster-admin",
+		},
+	}
+
+	_, err = roleBindingClient.Create(context.TODO(), roleBinding, metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (h *Helm) InstallCharts() error {
+	return h.runHelm([]string{"install", "--values", "/apps/config/airy.yaml", "core", "/apps/helm-chart/"})
```

Comment:
this airy.yaml is not "our official" config file right? Because if it is, it's confusing, if it isn't then the name is confusing :sorry:

Reply:
What do you mean by "official" config file? 🤔 Currently, we use the `airy.yaml` as the helm values file so it felt naturally to call the mount the same.

Reply:
we're embedding the default config in the CLI so I'm not exactly sure what's this file for

Reply:
I see: This is that default file and we mount it in the job's container using a config map. So I used `airy.yaml` as a generic name for that config. It's not the one from the infrastructure directory or any other file.